homeassistant/components/flux_led/__init__.py

```diff
@@ -1 +1,33 @@
 """The flux_led component."""
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["light"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Flux LED/MagicLight component."""
+    hass.data.setdefault(DOMAIN, {})
+
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Flux LED/MagicLight from a config entry."""
+    conf = entry.data
+
+    hass.data[DOMAIN][entry.entry_id] = {
```

Comment:
We don't need to store the entry data. The entry data is forwarded to the platform so the platform can access the entry data via the entry.

Reply:
OK - removed and changed light.py to pull from entry.data.